bdr-veeam/veeam-configure-local-backup-jobs.ps1

```
@@ -0,0 +1,242 @@
+    if ($env:RMM_SCRIPT_PATH) {
+        $LOG_DIR = "$env:RMM_SCRIPT_PATH\logs"
+        if (-not (Test-Path $LOG_DIR)) { New-Item -ItemType Directory -Path $LOG_DIR -Force | Out-Null }
+        $LOG_PATH = "$LOG_DIR\$SCRIPT_LOG_NAME"
+    } else {
+        $LOG_PATH = "$env:WINDIR\logs\$SCRIPT_LOG_NAME"
+    }
+}
+
+Start-Transcript -Path $LOG_PATH
+
+Write-Host "=== Veeam Local Backup Schedule Configuration ==="
```

PSScriptAnalyzer (1.24.0)

[warning] File 'veeam-configure-local-backup-jobs.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)